pkg/scalers/aws_sqs_queue_test.go

```diff
@@ -117,6 +117,17 @@ var testAWSSQSMetadata = []parseAWSSQSMetadataTestData{
 		},
 		false,
 		"with AWS Role from TriggerAuthentication"},
+	{map[string]string{
+		"queueURL":    testAWSSQSProperQueueURL,
+		"queueLength": "1",
+		"awsRegion":   "eu-west-1",
+		"podIdentity": "false"},
```

Comment:
I'm not a fan of this name given you can already connect without pod identity, what if we rename it to `identityOwner: pod|operator` or so?

Reply:
updated